helm_deploy/values-dev.yaml

```diff
@@ -16,7 +16,7 @@ ingress:
 
 env:
   JAVA_OPTS: "-Xmx512m"
-  HMPPSAUTH_BASEURL: "https://sign-in-dev.hmpps.service.justice.gov.uk"
+  HMPPSAUTH_BASEURL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   INTERVENTIONSUI_BASEURL: "https://hmpps-interventions-ui-dev.apps.live-1.cloud-platform.service.justice.gov.uk"
   COMMUNITYAPI_BASEURL: "https://community-api-secure.test.delius.probation.hmpps.dsd.io"
```

Comment:
Just thinking about how the syntax for `HMPPSAUTH_BASEURL` becomes our "configuration interface" to the outside world:

- env config in random `docker-compose.yml`s
- `docker run -e HMPPSAUTH_BASEURL=... service`
- anyone running it somehow

will have to know that the URL should contain the `/auth` postfix.

Is that an ok assumption to make? Will other apps also configure HMPPS auth with the `/auth` postfix?

Reply:
yes good point.